### ISSUE TITLE
Add convenience methods for group memberships

### ISSUE THREAD

**Issue by rdegges** [stormpath/stormpath-sdk-node#123]
We should add `Account.removeGroup()` and `Group.removeAccount()` stuff to make things nicer for users =)

**Comment by luyizhao** [comment]
+1. Very useful if you're using webhooks to manage account groups.

**Comment by robertjd** [comment]
Agreed!  Hey @luyizhao, I'd love to hear how you're using webhooks with Stormpath!

**Comment by luyizhao** [comment]
Mainly Stripe events which dictate the groups users should live in. Updating ```customData``` upon events is a decent workaround, although sometimes it's still better to roll your own ```removeGroup()``` functions to bypass caching with ```customData``` :) ... (Sidenote, interested in seeing how future cache-handling for groups develops as referenced [here](https://github.com/stormpath/stormpath-sdk-node/issues/143)!)

**Comment by robertjd** [comment]
I agree that we should add these methods, but what should the inputs be to these functions?  An already-constructed `Account` or `Group` object?  That makes the most sense, but what if I have just an href or name?  We have to define these different options
